### ISSUE TITLE
how to solve pdfviewer runtime library error?

### ISSUE THREAD

**Issue by ahmeedev** [syncfusion/flutter-widgets#1291]
After viewing multiple times the same file in pdf viewer, the program stuck and runtime error shows like this

![image](https://github.com/syncfusion/flutter-widgets/assets/75003816/3e289999-274f-4c6a-9d18-7368d6c0b7c3)

**Comment by deepika134** [comment]
We tried to reproduce the issue with the provided details, but PDF loaded properly without any exceptions on our side. For your reference, we have attached the sample and the output screen recording of the same. It can be downloaded from the attachment. If you are still facing the issue, kindly share the issue reproducing in the sample and also share the following details:

•	Syncfusion_pdf_viewer version.
•	Platforms on which you are facing the issue.
•	Flutter doctor log.
•	Device specifications.
•	PDF document on which you are facing the issue.
•	Issue reproducing video/steps.

This will be helpful for us to provide a prompt solution at the earliest.
[github_1291.zip](https://github.com/syncfusion/flutter-widgets/files/11998543/github_1291.zip)
